### PR TITLE
trvial-builders.nix: nixpkgs-fmt

### PR DESCRIPTION
**Pull request by happysalada**
###### Motivation for this change

just running the formatter on a file. I don't think this will result in any rebuild.
I think this is fairly uncontroversial, but leaving this open for 1 day in case somebody wants to have a look

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/build-support/trivial-builders.nix [hunk 1]
```diff
@@ -572,49 +576,52 @@ rec {
...
+    * };
+  */
   testVersion =
-    { package,
-      command ? "${package.meta.mainProgram or package.pname or package.name} --version",
-      version ? package.version,
+    { package
+    , command ? "${package.meta.mainProgram or package.pname or package.name} --version"
+    , version ? package.version
+    ,
```
Comment (SuperSandro2000) [inline pkgs/build-support/trivial-builders.nix:624]:
```suggestion
```
that is extra and shouldn't cause any rebuilds.

**Comment by roberth** [comment]
https://github.com/NixOS/nixpkgs/pull/128467 may conflict with this PR and should take priority. Maybe try again when staging-next has been merged. At that time, also fix the typo in the commit message and PR title.

**Comment by happysalada** [comment]
@roberth thanks for letting me know!

**Comment by vcunat** [comment]
The conflicting change has been merged.

**Comment by happysalada** [comment]
I messed up my branch, going to do this in another PR

**Comment by happysalada** [comment]
did this properly in https://github.com/NixOS/nixpkgs/pull/129392

**Comment by happysalada** [comment]
@vcunat thank you for letting me know about the merge!

**Pull request closed without merging**